test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Check if the yml exists
+if [[ ! -f $file ]]; then
+  echo "File $file doesn't exist!"
+  exit 1
+fi
+
+# Run test environment
+docker-compose -p ds -f $file up -d
+
+wakeup_attempts=30
+wakeup_timeout=5
+
+for ((i=0; i<$wakeup_attempts; i++))
+do
```

Comment:
Maybe remove those attempts and just add 30 second timeout?
In my practice 30 second is enough in 99% cases for DocumentServer to start
It's less precise but simpler to understand